Gopkg.lock

```diff
@@ -825,6 +825,7 @@
   analyzer-name = "dep"
   analyzer-version = 1
   input-imports = [
+    "github.com/ghodss/yaml",
```

Comment:
This does not cause any build failure since `ghodss/yaml` package was already in `vendor` folder ( before this PR ). In this PR, `dep` added it to the `input-imports` list as we have started using `hodss/yaml`.

Ideally, the unit test(s), vendoring, should accompany the code changes in the same commit to make it a cohesive unit. I am happy to merge them into one. I split them to avoid having a large commit.

Reply:
A general rule of thumb... Separating code changes and unit tests shouldn't be done to make commits smaller. Rather what is being introduced in the code should be logically split up if possible.